config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get "/customer/:id", to: "customer#show"
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
The route for the form would be: `post "/customers/:id/items", to: "customer_items#create"`